src/enums.rs

```diff
@@ -392,6 +528,50 @@ pub enum ProjectUrl {
     Unknown(Url),
 }
 
+impl Serialize for ProjectUrl {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        let mut s = serializer.serialize_struct("url", 2)?;
+        match self {
+            ProjectUrl::Donation(url) => {
+                s.serialize_field("type", "donation")?;
+                s.serialize_field("url", &url)?;
```

Comment:
You can do s.serialize_filed("url", &self.0)?; only once for all the project urls

Reply:
```
error[E0609]: no field `0` on type `&enums::ProjectUrl`
   --> src/enums.rs:571:40
    |
571 |         s.serialize_field("url", &self.0)?;
    |                                        ^

```